src/nxdoc/nuxeo-server/upgrading-the-nuxeo-platform/upgrade-from-lts-2019-to-cloud-2020.md

```diff
@@ -344,6 +344,9 @@ Here are a few breaking changes to take into account when upgrading from Elastic
 
   The field needs to be of `text` type.
 
+
+If you have to adapt your setting or mapping, you will need to re-index the repository to apply the change.
```

Comment:
Maybe link to the re-index page as below?

Reply:
at this stage (application adaptation) this is just a warning, the procedure is given when it is time to upgrade the elastic cluster (next section)